src/firefly/js/templates/lightcurve/LcConverterFactory.js

```diff
@@ -122,7 +124,7 @@ const converters = {
         converterId: 'ptf',
         defaultImageCount: 5,
         defaultTimeCName: 'obsmjd',
-        defaultYCname: 'mag_autocorr',
+        defaultYCname: 'magerr_auto',
```

Comment:
This must have being temporary  change when debug the Cname matching. should change back to "mag_autocorr"

Reply:
oh, you are right. Can you change it before merging?